tools/cli/go-whisk-cli/commands/action.go

```diff
@@ -556,14 +556,28 @@ func parseAction(cmd *cobra.Command, args []string) (*whisk.Action, bool, error)
         return nil, sharedSet, err
       }
     } else {
-      whisk.Debug(whisk.DbgError, "Action runtime extension '%s' is not supported\n", ext)
-      errMsg := fmt.Sprintf(
-        wski18n.T("'{{.name}}' is not a supported action runtime",
-          map[string]interface{}{"name": ext}))
+      errMsg := ""
```

Comment:
Minor comment.
```
    } else {
      errMsg := ""
      if ext == ".zip" {
```
No need for the `errMsg := ""` line.  Just add the `:=` when the errMsg is being created.  Then the `if` statement can be directly joined to the `else` following the existing `else if` flow.